notebooks/COS/Extract/Extract.ipynb

```diff
@@ -0,0 +1,1580 @@
+    "- For a detailed overview of the COS instrument, see the [COS Instrument Handbook](https://hst-docs.stsci.edu/display/COSIHB/).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "sunset-oliver",
+   "metadata": {},
+   "source": [
+    "## We will import the following packages:\n",
+    "\n",
+    "- `numpy` to handle arrays and functions\n",
+    "- `astropy.io fits` and `astropy.table Table` for accessing FITS files\n",
+    "- `glob`, `os`, and `shutil` for working with system files\n",
+    "- `astroquery.mast Observations` for finding and downloading data from the [MAST](https://mast.stsci.edu/portal/Mashup/Clients/Mast/Portal.html) archive\n",
+    "- `matplotlib.pyplot` for plotting\n",
+    "- `matplotlib.image` for reading in images\n",
+    "- `calcos` to run the CalCOS pipeline for COS data reduction\n",
+    "- `scipy.interpolate interp1d` for interpolating datasets to the same sampling\n",
+    "\n",
+    "These python packages are installed standard with the the STScI conda distribution. For more information, see our notebook tutorial on [setting up an environment](https://github.com/spacetelescope/COS-Notebooks/blob/master/Setup/Setup.ipynb).\n",
+    "\n",
+    "We'll also filter out two unhelpful warnings about a deprecation and dividing by zero which do not affect our data processing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "municipal-scottish",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import For: array manipulation\n",
+    "import numpy as np\n",
+    "# Import For: reading fits files\n",
+    "from astropy.table import Table\n",
+    "from astropy.io import fits\n",
+    "\n",
+    "# Import For: downloading the data\n",
+    "from astroquery.mast import Observations as Obs\n",
+    "\n",
+    "# Import For: Plotting\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "# Import For: showing images from within Python\n",
+    "from matplotlib import image as mpimg\n",
+    "\n",
+    "# Import For: dealing with system files\n",
+    "import glob, os, shutil\n",
+    "\n",
+    "# Import For: running the CalCOS pipeline\n",
+    "import calcos\n",
+    "\n",
+    "# Import For: comparing the old and new CalCOS values\n",
+    "from scipy.interpolate import interp1d \n",
+    "\n",
+    "# We will also suppress a warning that won't affect our data processing:\n",
+    "np.seterr(divide = 'ignore') \n",
+    "np.warnings.filterwarnings('ignore', category=np.VisibleDeprecationWarning)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "banner-hypothesis",
+   "metadata": {},
+   "source": [
+    "## We will also define a few directories we will need:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "psychological-camel",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# These will be important directories for the notebook\n",
+    "datadir = './data/'\n",
```

Comment:
As with the other notebooks, should change to something cross-platform like `pathlib`